api/PluginArray.json

```diff
@@ -128,10 +128,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": true
+              "version_added": "10"
```

Comment:
A `length` property introduced much later than the interface. I don't believe it :)